src/mixins/datetime.js

```diff
@@ -92,7 +92,10 @@ export default {
          */
         datetimeFormat(value, format) {
             if (value !== undefined && value !== "") {
-                return dayjs.utc(value).tz(this.timezone).format(format);
+                const date = new Date(value);
+                return date.toLocaleString(this.$i18n.locale, { ...format,
```

Comment:
have you tested that this works?
Expanding a string like you are doing here is not likely to be correct.

![image](https://github.com/louislam/uptime-kuma/assets/26258709/a46c9d9b-99a0-4392-87d9-341f4a074756)

See line 84 of this usage